<!DOCTYPE html>
<html lang="en">
<head>
<title>mbiome</title>
<meta charset="UTF-8">
<meta name="viewport" content="width=device-width, initial-scale=1">
<link rel="stylesheet" href="css/w3.css">
<link rel="stylesheet" href="css/lato.css">
<link rel="stylesheet" href="css/montserrat.css">
<link rel="stylesheet" href="css/font-awesome.css">
<link rel="stylesheet" href="css/prism.css">
<script src="js/prism.js"></script>
<style>
body,h1,h2,h3,h4,h5,h6 {font-family: "Lato", sans-serif}
.w3-bar,h1,button {font-family: "Montserrat", sans-serif}
.fa-anchor,.fa-coffee {font-size:200px}
</style>
</head>
<body>

<!-- Navbar -->
<div class="w3-top">
  <div class="w3-bar w3-grey w3-card w3-left-align w3-large">
    <a href="#" class="w3-bar-item w3-button w3-padding-large w3-white">home</a>
  </div>
</div>

<!-- Header -->
<header class="w3-container w3-blue-grey w3-center" style="padding:128px 16px">
  <h1 class="w3-margin w3-jumbo">microbial communities</h1>
  <p class="w3-xlarge">bacterial ecology and human health</p>
</header>

<!-- First Grid -->
<div class="w3-row-padding w3-padding-64 w3-container">
<div class="w3-content">
<div class="w3-threequarter">

<h1>the "microbiome" and human health</h1>

<h5 class="w3-padding-32 w3-text-grey">
  Bacteria impact our health, and not always in a bad way.
</h5>

<p>
  When we think about bacteria and human health, the mind almost always
  turns to pathogens, first. While pathogenic microbes certainly have
  strong impacts on health, microbial communities can produce much more
  subtle effects that contribute to health and disease.
</p>

<p>
  In general, science has a healthy appreciation for the many roles
  microbial communities have and continue to play on earth. It is widely
  appreciated that the evolution of microbes has gone hand-in-hand with
  broad changes in earth bio-geo-chemistry; microbes have shaped the planet
  over billions of years. Microbial communities continue to play key roles
  in global chemical and nutrient cycling and contribute to weather and
  climate processes on a global scale.
</p>

<p>
  Microbial communities also play key roles in plant and animal health.
  Of course, the most 'obvious' impact of microbes on plant and animal
  health is as causes of pathogenic disease. But it's also safe to say
  that plant and animal biology would not be the same without associated
  microbial communities. Soil and root-associated microbes play key roles
  in plant metabolism, and gut-associated microbes are important for
  digestion and gut health in humans and other animals.
</p>

<p>
  While it is obvious that microbial communities are critically important
  for the proper functioning of 'higher' organisms, and for the functioning
  of earth's biosphere, what is less well understood is precisely <em>how</em>
  microbial communities impact these larger processes.
</p>

<p>
  One of the central goals of the field of 'microbial ecology' is to better
  understand how the structure and dynamics of microbial communities
  impacts the functioning of 'larger scale' processes.
</p>

<p>
  In the case of microbial pathogens, the 'connection' between the microbial
  community and the 'larger scale' process of human disease can be
  relatively simple: if you 'get' the pathogen, you 'get' the disease.
</p>

<p>
  In other cases, however, the 'connection' between the microbial community
  and a complex human disease can be much more complex.
</p>

<p>
  For example, it is now widely appreciated that the community of microbes
  living in the human gut plays a key role in the development of the human
  immune system, probably via direct interactions with the gut epithelial
  cells during early development. If these interactions work 'correctly',
  the human immune system develops 'appropriately'. However, if something
  goes 'wrong' with this process, the human immune system can develop
  'incorrectly', leading to a wide variety of autoimmune disorders.
</p>

<p>
  An example of a human autoimmune disorder associated with the gut microbial
  community is type 1 diabetes (T1D). The 'cause' of type 1 diabetes is
  destruction of insulin-producing B-cells in the pancreas by the body's own
  immune system. Once B-cells are destroyed by autoimmunity, the body can
  no longer produce insulin, which is required for human life. At this point,
  the individual is completely dependent on exogenous insulin.
</p>

<p>
  In contrast to the much more prevalent type 2 diabetes (which has an entirely
  different cause), type 1 diabetes primarily effects children. In additoin to
  genetic and environmental risk factors, it is thought
  that the structure of the gut microbial community during early childhood
  may have a strong impact on an individual's risk for developing T1D.
  However, exactly <em>how</em> the gut microbial community may contribute to
  T1D risk is not understood.
</p>

<p>
  There is a robust international research effort aimed at characterizing
  the structure and dynamics of the human gut-associated microbial community
  in 'healthy' subjects, and comparing 'healthy' gut microbial communities
  to those found in subjects who will develop type 1 diabetes, in order to
  better understand how changes in the structure and dynamics of the gut
  "microbiome" might impact disease risk, onset and progression.
</p>

<p>
  This is only one example of how researchers are attempting to untangle how
  microbial ecology might causally impact larger-scale processes. Whatever the
  particular larger-scale process under study, the structure of the research
  program is very similar: characterize the structure of the microbial
  community, and then determine which changes in microbial community structure
  are associated with observed differences in the larger-scale process.
</p>

</div>
<div class="w3-quarter w3-center">
<i class="fa fa-thermometer-2 fa-5x w3-padding-64 w3-text-blue w3-margin-right"></i>
</div>
</div>
</div>

<!-- Second Grid -->
<div class="w3-row-padding w3-light-grey w3-padding-64 w3-container">
<div class="w3-content">
<div class="w3-quarter w3-center">
<i class="fa fa-bar-chart fa-5x w3-padding-64 w3-text-blue w3-margin-right"></i>
</div>

<div class="w3-threequarter">

<h1>characterizing microbial communities</h1>

<h5 class="w3-padding-32 w3-text-grey">
  The 16S rDNA paradigm.
</h5>

<p>
  Scientists have come up with multiple approaches for characterizing the
  structure of microbial communities. One of the most commonly used methods
  relies on amplifying and sequencing a 'universal' region of the bacterial
  genome, which can be used to organize and compare bacterial 'species'
  within a sample.
</p>

<p>
  The specific 'universal marker' most commonly used in bacterial community
  sequencing is a specific region of the 16S rDNA gene, which encodes a
  component of the bacterial ribosome. The characteristics of this particular
  'marker' that make it 'good' for characterizing bacterial community
  composition are:

  <ul>
    <li>It is 'universal' - ie, the 16S rDNA gene is found in (nearly?) all bacterial species.</li>
    <li>It provides good 'resolution' - nearly all unique bacterial 'species' can be identified using the 16S rDNA sequence, and the sequence is not <em>too</em> different to prevent comparisions among very distantly-related bacteria.</li>
  </ul>
</p>

<p>
  In the absence of morphological, genomic or other data, sequencing a specific
  'universal' region of the bacterial 16S rDNA gene allows researchers to
  fairly confidently identify particular bacterial 'species' in a biological
  sample. In other words, 16S rDNA sequencing provides a 'marker' that can
  be used to 'cluster' bacteria in a sample into different 'species' (or
  some other taxonomic level appropriate for the specific study goals).
</p>

<p>
  It's important to note that no particular 'marker' is perfect, and there is
  much debate concerning the utility of the 16S rDNA sequencing approach,
  both in general and in particular use cases. Scientific debate notwithstanding,
  it's pretty safe to say that the vast majority of existing bacterial
  community studies use a 16S rDNA sequencing approach, and this approach is
  likely to dominate the majority of future studies, at least for a few
  more years.
</p>

<p>
  It's also important to note that 16S rDNA sequencing provides researchers
  with an estimate of 'which' particular 'species' of bacteria is present
  in a sample, but not necessarily with information about what that bacterium
  is actually <em>doing</em> in the community. For this reason, 16S rDNA
  sequencing studies are confined to characterizing the <em>species composition</em>
  of the microbial community; they cannot directly investigate the
  <em>functioning</em> of bacterial communities or their individual
  components.
</p>

<p>
  The information or data you get out of a 16S rDNA sequencing study are a
  'list' of bacterial 'species' and the relative abundance of each species
  in a particular biological sample. In practice, the 'species' in a sample
  may or may not correspond to the Linnaean classification system. Depending
  on the study design, you might get genus-species 'names' for the bacteria
  in the sample, or you might just get 'operational taxonomic units', with
  no clear connection to known "species".
</p>

<p>
  In any case, the 16S rDNA analysis
  provides you with <em>which</em> species are present in a sample, as well
  as <em>how many</em> (relatively!) of each species is present. From this
  information, you can determine who the 'dominant players' are in a given
  sample's bacterial community, as well as which bacteria might also be
  present, but in lower numbers. By comparing the species compositions across
  different samples, scientists can start to better understand how microbial
  community composition changes in response to different environmental
  'features', such as time, nutrient levels, or disease/health of the
  'host' species.
</p>

<p>
  Recently, there has been quite a bit of research interest in the idea that
  we might be able to use bacterial species composition to help 'predict'
  an individual's disease risk. For example, if the human gut microbial
  community impacts type 1 diabetes risk, could we sequence an individual's
  gut microbial community in early childhood, and then use this information
  to better predict the individual's risk of later developing T1D? If we
  could reliably predict T1D risk early, could we develop and deploy
  interventions to help prevent later development of the disease?
</p>

<p>
  Similar to the way traditional genomic phenotype 'prediction' methods work,
  traditional approaches used to study how bacterial communities impact
  larger-scale processes have largely consisted of examining the statistical
  association between <em>each individual</em> microbial taxon and the
  process under investigation, and then finding <em>which particular</em>
  microbial taxa are more strongly associated with observed differences
  in the larger-scale process than would be expected, based on chance.
</p>

<p>
  These types of simple 'association' studies are well-suited to identifying
  particular strongly-associated bacterial 'species', which can then be
  isolated and studied in the lab.
</p>

<p>
  But what if the association between the gut microbial community and disease
  risk is not simple or driven by a relatively small number of individual
  'species'? If the 'real' causal links between the gut microbial community
  and disease risk are complex and involve interactions among a relatively
  large number of 'species', they could easily be missed by traditional
  approaches that only consider each species in isolation.
</p>

<p>
  To address this issue, researchers are turning to increasingly complex
  statistical modeling approaches, like deep neural networks, to attempt to
  'capture' complex nonlinear interactions among bacterial species that
  could impact larger-scale processes like autoimmune disease risk.
</p>

<p>
  While complex statistical modeling approaches have the potential to shed new
  light on how microbial community structure impacts its function in the
  environment, these methods are known to require very large training data
  sets to be reliable. And although 16S rDNA sequencing costs continue to
  fall dramatically, <em>sampling</em> microbial communities - whether in
  the deep ocean or in the human gut - remains difficult and expensive at
  the scale needed to reliably train deep neural networks.
</p>

</div>
</div>
</div>


<div class="w3-container w3-black w3-center w3-opacity w3-padding-64">
    <h1 class="w3-margin w3-xlarge">what you have learned == what you can do</h1>
</div>

<!-- Footer -->
<footer class="w3-container w3-padding-64 w3-center w3-opacity">
  <div class="w3-xlarge w3-padding-32">
    <p>end.</p>
 </div>
</footer>

</body>
</html>
